critiquebrainz/frontend/external/musicbrainz_db/place.py

```diff
@@ -23,28 +23,42 @@ def get_place_by_id(mbid):
     key = cache.gen_key(mbid)
     place = cache.get(key)
     if not place:
-        place = _get_place_by_id(
-            mbid, includes=['artist-rels', 'place-rels', 'release-group-rels', 'url-rels'],
-        )
+        place = fetch_multiple_places(
+            mbids=[mbid],
+            includes=['artist-rels', 'place-rels', 'release-group-rels', 'url-rels'],
+        ).get(mbid)
     cache.set(key=key, val=place, time=DEFAULT_CACHE_EXPIRATION)
     return place
 
 
-def _get_place_by_id(place_id, includes=[]):
+def fetch_multiple_places(*, mbids, includes=[]):
```

Comment:
I am not very familiar with the code, but default mutable values for function arguments (like `includes=[]` here) can cause problems. https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html